javascript/data.js

```diff
@@ -0,0 +1,63 @@
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum',
+  liveBtn: 'https://google.com',
+  sourceBtn: 'https://google.com',
+};
+
+const project2 = {
+  id: 2,
+  featureImg: 'project-img2.svg',
+  projectName: 'Project 2',
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsumg',
+  liveBtn: 'https://youtube.com',
+  sourceBtn: 'https://youtube.com',
+};
+
+const project3 = {
+  id: 3,
+  featureImg: 'project-img3.svg',
+  projectName: 'Project 3',
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum',
+  liveBtn: 'https://google.com',
+  sourceBtn: 'https://google.com',
+};
+
+const project4 = {
+  id: 4,
+  featureImg: 'project-img4.svg',
+  projectName: 'Project 4',
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum',
+  liveBtn: 'https://google.com',
+  sourceBtn: 'https://google.com',
+};
+
+const project5 = {
+  id: 5,
+  featureImg: 'project-img5.svg',
+  projectName: 'Project 5',
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum',
+  liveBtn: 'https://google.com',
+  sourceBtn: 'https://google.com',
+};
+
+const project6 = {
+  id: 6,
+  featureImg: 'project-img6.svg',
+  projectName: 'Project 6',
+  technologies: ['Ruby on Rails', 'CSS', 'HTML', 'Javascript'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsu',
+  liveBtn: 'https://google.com',
+  sourceBtn: 'https://google.com',
+};
+
+const projectInfo = [project1, project2, project3, project4, project5, project6];
```

Comment:
- [ ] For best practices please consider adding the objects directly inside the array